src/System.Security.Cryptography.OpenSsl/pkg/System.Security.Cryptography.OpenSsl.pkgproj

```diff
@@ -3,12 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Security.Cryptography.OpenSsl.csproj">
-      <!-- Removing net461 and AllXamarianFrameworks from validation because of https://github.com/dotnet/corefx/issues/25924 -->
```

Comment:
Do we not need the xamarin framework validation to be added eventually anymore?

Reply:
I don't think so given that this package doesn't have any xamarin assets and consumes need to consume the binary from this package in order to use it there.